### ISSUE TITLE
Directive inside uib-tab not being rendered correctly

### ISSUE THREAD

**Issue by cdparra** [angular-ui/bootstrap#4872]
I am trying to find the source of an issue with a third-party directive that appears when I use that directive inside a uib-tab. The third-party directive is [angular-multi-slider][1] and the original issue was first reported in [that repository][2]. An use case is available [here][3]. Click on the second tab, and you will see that the inner slider has all its handles one on top of the others (i.e., widths=0px). Then click on on of the handles and it will appear correctly. 

version of AngularJS = 1.4.7
version of this library that you are using = 0.14.3
3rd-party libraries used, if any = [angular-multi-slider][1] 0.1.1

Basically, as you can see, the inner directive (the slider) is rendered correctly under the first tab. Under the second tab, however, I have to first click on it for it to be correctly rendered.

The issue persists even after following your recommendation regarding scopes in the [FAQ][4]

[1]: https://github.com/enkodellc/angular-multi-slider/
[2]: https://github.com/enkodellc/angular-multi-slider/issues/1#issuecomment-155534915
[3]: http://plnkr.co/edit/dQaJAJzOcKOKA18m25hW?p=preview
[4]: https://github.com/angular-ui/bootstrap/wiki/FAQ#my-input--select-element-doesnt-work-as-expected-inside-the-tab--accordion-directive

**Comment by icfantv** [comment]
@cdparra, please do not use the issues forum for support related-requests.  Rather, please follow the instructions [here](https://github.com/angular-ui/bootstrap#need-help).  The issues forum is reserved for bugs.

That said, I've experienced a bit of growth pains w/r/t tabs and such and can try to help out as best I can.  Please post your question to SO as documented above and give me the link.  Also create a minimally working plunker that demonstrates the issue.

**Comment by cdparra** [comment]
Adding the [SO question](http://stackoverflow.com/questions/33642247/how-to-make-other-directives-work-inside-uib-tab-elements). Thanks!
